appcenter-crashes/android/src/main/java/com/microsoft/appcenter/reactnative/crashes/AppCenterReactNativeCrashesModule.java

```diff
@@ -183,7 +183,10 @@ public void sendErrorAttachments(ReadableArray attachments, String errorId) {
             Collection<ErrorAttachmentLog> attachmentLogs = new LinkedList<>();
             for (int i = 0; i < attachments.size(); i++) {
                 ReadableMap jsAttachment = attachments.getMap(i);
-                String fileName = jsAttachment.getString("fileName");
+                String fileName = null;
+                if (jsAttachment.hasKey("fileName")) {
```

Comment:
We should make `fileName`, `text`, `data` and `contentType` as constant (static final in Java).